lib/engine/game/g_18_india/game.rb

```diff
@@ -233,9 +231,15 @@ def init_corporations(stock_market)
           end
         end
 
-        # Include bond shares for cache_objects
+        # Include IPO Pool shares and bond shares for cache_objects
         def shares
-          @corporations.flat_map(&:shares) + @players.flat_map(&:shares) + @share_pool.shares + gipr.bond_shares
+          LOGGER.debug "@corporations.flat_map(&:shares) => #{@corporations.flat_map(&:shares)}"
```

Comment:
One note on these `LOGGER` calls, and I think I missed this in a prior PR, but the interpolated string should be passed in a block. This defers evaluating the string, meaning in environments where the string is not going to actually get logged, it never gets evaluated. I put some more info about this in #10946, and if we pull in my custom rubocop plugin you'd be able to have rubocop autocorrect these for you. So consider it optional to change them for now.

```suggestion
          LOGGER.debug { "@corporations.flat_map(&:shares) => #{@corporations.flat_map(&:shares)}" }
```

Reply:
Many of these LOGGER statements are just for me to get some state information while I was tracking down a bug.  Or simply verify result of some code. 

How do I pull in your plugin?  I'll gladly auto correct them.

Reply:
I have updated the LOGGER calls in this file.  I can also do the other, or wait to try using the RuboCop to auto fix them.

Reply:
To pull in the plugin you'd add it to your `Gemfile`, as done here - https://github.com/tobymao/18xx/pull/10946/files#diff-d09ea66f8227784ff4393d88a19836f321c915ae10031d16c93d67e6283ab55f

Then run `bundle install` in your container (or simply kill the container and run `make dev_up_b` to rebuild it)

I think it's fine to wait until that's merged in. Thanks for updating the calls here!